Kubernetes/omsagent.yaml

```diff
@@ -345,6 +372,8 @@ spec:
           name: host-log
         - mountPath: /var/lib/docker/containers 
           name: containerlog-path
+        - mountPath: /etc/kubernetes
```

Comment:
please mount anything from host into <whateveryoumount>/host. ex;- in this case it would be /etc/kubernetes/host.